src/gifcodec.c

```diff
@@ -207,7 +207,7 @@ DGifSlurpMono(GifFileType * GifFile, SavedImage *TrailingExtensions)
 				if (ImageSize == 0)
 					return GIF_ERROR;
 
-				sp->RasterBits = (BYTE*) GdipAlloc (ImageSize * sizeof (GifPixelType));
+				sp->RasterBits = (BYTE*) malloc (ImageSize * sizeof (GifPixelType));
```

Comment:
This doesn't make sense to me, GdpiAlloc is just a wrapper around malloc:

https://github.com/mono/libgdiplus/blob/63aed45cc6d4c10c02cb02e3cd1256d0317ddf93/src/general.c#L72-L76